test/parallel/test-tls-client-allow-partial-trust-chain.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
```

Comment:
Since this is using the `node:test` syntax, this could likely use the `skip: !common.hasCrypto` option in the test definition itself and just move the `require('tls')` into the test body.

Reply:
i think this is okay. `common.skip` will mark the entire test as skipped, while the test runner's skip will only mark the tests as skip, but the file as passed.

Reply:
Done in 509156ac75aed153b3deab72b33eec6c3e6efe17 👍

Reply:
Sorry, did this before seeing @RedYetiDev's response – no strong feelings from my side.